ClosedXML_Tests/Excel/CalcEngine/FunctionsTests.cs

```diff
@@ -1,10 +1,9 @@
-using System;
 using ClosedXML.Excel;
 using NUnit.Framework;
+using System;
```

Comment:
I realise CodeMaid fixed the whitespace issues, but as this file was essentially unchanged (logically), all whitespace changes should also be reverted. Just revert the entire file.

Reply:
Done